.github/workflows/ubuntu.yml

```diff
@@ -6,19 +6,18 @@ jobs:
   build:
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
-        ruby: [ '2.6', '2.5', '2.4', 'ruby-head']
+        ruby: [ ruby-head, 2.7, 2.6, 2.5, 2.4 ]
     steps:
-    - name: Install libraries
-      run: sudo apt install haveged
-    - uses: actions/checkout@master
-    - name: Set up Ruby
-      uses: ruby/setup-ruby@v1
-      with:
-        ruby-version: ${{ matrix.ruby }}
-    - name: Install dependencies
-      run: |
-        gem install bundler --no-document
-        bundle install
-    - name: Run test
-      run: rake
+      - name: Install libraries
+        run: sudo apt install haveged
+      - uses: actions/checkout@v2
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+      - name: Install dependencies
+        run: bundle install
+      - name: Run test
+        run: rake
```

Comment:
The indentation changes here make it harder to read the diff, could you keep the original indentation?

Reply:
I could, but it was my impression that proper yaml should have array members indented...

Reply:
Yeah, I'm not sure if there is a recommendation on that, I think both styles were used in `.travis.yml` back then.
I personally prefer the lesser indented variant, so it's only 4-spaces indented for steps (6 starts to feel quite deep already).

Reply:
Re indenting, not an issue I feel strongly about.  But, since I changed the indents in `.github/workflows/ubuntu-jruby.yml`, knowing the preference affects what I do with the merge conflict.

My tired old eyes prefer the indented style...